addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderBaseThingHandler.java

```diff
@@ -383,25 +437,28 @@ protected State getChannelBoolAsOnOffState(boolean value) {
             }
 
         } catch (Exception ex) {
```

Comment:
Please catch specific exceptions instead of catching all exceptions. That way it is more clear what exceptions need to be handled and how this is done.